models/article.models.js

```diff
@@ -0,0 +1,10 @@
+const db = require('../db/connection');
+
+exports.selectArticlesById = (article_id) => {
+   return db.query( 
+    `SELECT * FROM articles
+     WHERE article_id = $1`,
+     [article_id])
+     .then((result) => {
+        return result.rows[0]})
```

Comment:
Here we need to add a  custom error to handle the 404.
https://notes.northcoders.com/courses/js-back-end/error-handling#custom-errors